### PR TITLE
Benchmark framework for ATS

### PR DESCRIPTION
**Pull request by bryancall**

### CODE REVIEW

**Comment by zwoop** [comment]
Do we really need to check in all these copies of e.g. the default body factory again? Couldn't we have a "setup" script that takes the necessary files from an install directory instead, before running the benchmark? Similar to how autest takes a "install" directory as a bas for setting up a test.

File: tools/benchmark/etc/trafficserver/remap.config [hunk 1]
```diff
@@ -0,0 +1 @@
+map / http://192.168.1.10/
```
Comment (SolidWallOfCode) [inline tools/benchmark/etc/trafficserver/remap.config]: What is that's not a valid address?

File: tools/benchmark/etc/trafficserver/header_rewrite.config [hunk 2]
```diff
@@ -0,0 +1,5 @@
+cond %{READ_REQUEST_PRE_REMAP_HOOK} [AND]
```
Comment (SolidWallOfCode) [inline tools/benchmark/etc/trafficserver/header_rewrite.config]: I can't find anywhere these URLs would be used. Neither can I find where `header_rewrite` would be used either.

**Comment by SolidWallOfCode** [comment]
I think the config files that contain only comments should be removed or compressed to a single line of "#".

**Comment by zwoop** [comment]
Yeh, I agree with @SolidWallOfCode, we should only duplicate files that are making changes. Or even better, have the benchmark script(s) modify the necessary configs accordingly.

Ideally, this benchmark tool should work similar to how autest works, where you specify the install directory of the ATS installation you want to benchmark, and then magic happens.

**Comment by bryancall** [comment]
[approve ci]

**Pull request closed without merging**